# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/jfsutils/hotfix-basename.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- jfsutils-1.1.15/fsck/fsck_message.c.vanilla	2024-07-25 12:54:38.270788213 +0200
+++ jfsutils-1.1.15/fsck/fsck_message.c	2024-07-25 12:54:49.339788079 +0200
@@ -3,6 +3,8 @@
 #include <stdio.h>
 #include <stdarg.h>
 #include <string.h>
+#include <libgen.h>
+
 #include "fsck_message.h"
 #include "fsckwsp.h"
 #include "jfs_endian.h"
--- jfsutils-1.1.15/fscklog/fscklog.c.vanilla	2024-07-25 12:55:03.036787914 +0200
+++ jfsutils-1.1.15/fscklog/fscklog.c	2024-07-25 12:55:10.849787820 +0200
@@ -70,6 +70,7 @@
 #include <stdarg.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <libgen.h>
 
 #include "jfs_version.h"
 #include "jfs_fscklog.h"
